member.go

```diff
@@ -22,11 +22,14 @@ import (
 )
 
 type Member struct {
-	client                   *Client
-	Id                       string   `json:"id"`
-	AvatarHash               string   `json:"avatarHash"`
-	Bio                      string   `json:"bio"`
-	BioData                  string   `json:"bioData"`
+	client     *Client
+	Id         string `json:"id"`
+	AvatarHash string `json:"avatarHash"`
+	Bio        string `json:"bio"`
+	BioData    struct {
+		Emoji struct {
+		} `json:"emoji"`
+	} `json:"bioData"`
```

Comment:
That's a good question. I included the empty struct because for my account, the JSON response returns an empty object.

The [API docs](https://developers.trello.com/advanced-reference/member#get-1-members-idmember-or-username) show no information about this, and I don't see anywhere in my Trello settings where I can set anything like this. I think it's related to Trello Gold since that allows you to upload custom emoji. I don't have that functionality, so I'm unsure what the format of that object would be. I emailed Trello support to ask them about it since their API docs are a bit light on explaining what expected results will be for many of their API responses.

If this is for custom emojis, my guess is that the key will be the keyword for the emoji, and the value will be the URL for the image file used for the emoji. I'm not sure how (or if it's possible) to define dynamic values in a struct in Go, so how would that be used for this struct?